### PR TITLE
Created time-drift.sh to check the time difference between systems

### PR DESCRIPTION
**Pull request by Reth11**
Created check-time-drift to validate the output of the time difference

### CODE REVIEW

File: scripts/time-drift.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
+for i in $(serviced host list| awk '{print $4}' | grep -v Name | grep -v $HOSTNAME); do echo ntpdate \-q $i; ntpdate -q $i; done
```
Comment (tgjonestx) [inline scripts/time-drift.sh]: I'm not sure we can assume DNS entries for all hosts, so checking by IP might be more reliable.   Also, 
* You can simplify a little by asking just for the fields you want, thus removing the need for awk/
* I think a multiline impl would be a little easier for readability.
* If by chance the last host in the list is unreachable, then the script exits with a non-zero exit code and inspector will skip the execution of `check-time-drift.py` with an error message like "one or more of its dependencies are not available"

So I suggest something like this:

```
HOST_IP=`hostname -i`
for i in $(serviced host list --show-fields Addr | grep -v ^Addr  | grep -v $HOST_IP); do
   echo ntpdate \-q $i
   ntpdate -q $i
done
exit 0
```

File: scripts/check-time-drift.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```
Comment (tgjonestx) [inline scripts/check-time-drift.py]: This file was added to git with 664 perms which means it's not executable by the time it's bundled into inspector and delivered to a customer site.  It should have 755 perms. Just chmod it and then `git add` it, and that should fix it.

File: scripts/time-drift.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
```
Comment (Reth11) [inline scripts/time-drift.sh]: Z has recommended based on several instances where ntp wasn't working or the time drift was too far apart between host, which was causing issues. I am wanting to run this against all the systems in the environment.

File: scripts/time-drift.sh (hunk 1 above)
Comment (Reth11) [inline scripts/time-drift.sh]: I agree, thank your for catching this and I have updated the code for 10 seconds.

Comment (Reth11) [inline scripts/time-drift.sh]: Thank you, 
I have updated the code to use the IP address for connectivity. 
I updated the code to filter out the host-name and that does look better.
I updated the code to use multiply lines and it is easier to read.
I updated the code to have an exit 0 of zero. 
I like your changes thank you.

File: scripts/check-time-drift.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+# zenoss-inspector-info
+# zenoss-inspector-tags time-drift verify
```
Comment (Reth11) [inline scripts/check-time-drift.py]: Thank you, I have made the change

File: scripts/check-time-drift.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    with open('time-drift.sh.stderr', 'r') as g:
+        elines = g.readlines()
+    # Check for errors
+    ehost = 0
+    for line in elines:
+        if 'no server suitable for synchronization found' in line:
+            ehost += 1
+            # The name of the host is not recorded during an error
+    if ehost > 0:
+        print 'WARN: %i host could not be reached for time-drift.' % ehost
```
Comment (Reth11) [inline scripts/check-time-drift.py]: Thank you for catching this.

File: scripts/check-time-drift.py (hunk 2 above)
Comment (Reth11) [inline scripts/check-time-drift.py]: Thank you, the change has been made

**Pull request merged**